database/connection.py

```diff
@@ -0,0 +1,15 @@
+import pymysql
+
+#database connection
+connection = pymysql.connect(host="localhost", user="root", passwd="", database="test")
+cursor = connection.cursor()
+select_tbl= "select * from tbl_employee "
+cursor.execute(select_tbl)
+
+
+
+#commiting the connection then closing it.
+connection.commit()
+connection.close()
+print("end");
+print("hello")
```

Comment:
Don't write two print statements. It can be done in one print statement.

Reply:
It can be done but we need to print both thing separately

Reply:
Can you please explain to me the reason?